code/game/objects/effects/decals/posters/bs12.dm

```diff
@@ -1,290 +1,67 @@
+DEFINE_POSTER(bay_47, "bsposter47", "Security pinup",                                  "This is a pin-up poster. A dark skinned white haired girl poses in the sunlight wearing a tank top with her stomach exposed. The text on the poster states \"M, Succubus of Security.\" and a lipstick mark stains the top right corner, as if kissed by the model herself.")
+DEFINE_POSTER(bay_48, "bsposter48", "Borg pinup?",                                     "This is a.. pin-up poster? It is a diagram on an old model of cyborg with a note scribbled in marker on the bottom, on the top there is a large XO written in red marker.")
+DEFINE_POSTER(bay_49, "bsposter49", "Engineering recruitment",                         "This is a poster showing an engineer relaxing by a computer, the text states \"Living the life! Join Engineering today!\"")
+DEFINE_POSTER(bay_50, "bsposter50", "Pinup Girl Cindy Kate",                           "This particular one is of Cindy Kate, a seductive performer well known among less savoury circles.")
+DEFINE_POSTER(bay_51, "bsposter51", "space appreciation poster",                       "This is a poster produced by the Generic Space Company, as a part of a series of commemorative posters on the wonders of space. One of three.")
+DEFINE_POSTER(bay_52, "bsposter52", "fire safety poster",                              "This is a poster reminding you of what you should do if you are on fire, or if you are at a dance party.")
+DEFINE_POSTER(bay_53, "bsposter53", "fire extinguisher poster",                        "This is a poster reminding you of what you should use to put out a fire.")
+DEFINE_POSTER(bay_54, "bsposter54", "firefighter poster",                              "This is a poster of a particularly stern looking firefighter. The caption reads, \"Only you can prevent space fires.\"")
+DEFINE_POSTER(bay_55, "bsposter55", "Earth appreciation poster",                       "This is a poster produced by the Generic Space Company, as a part of a series of commemorative posters on the wonders of space. Two of three.")
+DEFINE_POSTER(bay_56, "bsposter56", "Mars appreciation poster",                        "This is a poster produced by the Generic Space Company, as a part of a series of commemorative posters on the wonders of space. Three of three.")
+DEFINE_POSTER(bay_57, "bsposter57", "space carp warning poster",                       "This poster tells of the dangers of space carp infestations.")
+DEFINE_POSTER(bay_58, "bsposter58", "space carp information poster",                   "This poster showcases an old spacer saying on the dangers of migrant space carp.")
+DEFINE_POSTER(bay_59, "bsposter59", "poster - Miss Science 2299",                      "A large piece of space-resistant printed paper. This pin-up poster depicts a woman wearing a corporate labcoat, a bikini, and a sheepish grin. She's shyly posing atop some highly specialized research equipment.")
+
+#undef DEFINE_POSTER
```

Comment:
👎 I don't like this, personally. The define should be available for maps and modpacks to use.

Reply:
I could always move it to the define folder and make it global if there's interest for it. 
But good practice dictates not to make macro available outside of where you need them as much as possible. Which is why I didn't make it a global macro to start with.